app-admin/doctl/doctl-9999.ebuild

```diff
@@ -1,19 +1,17 @@
-# Copyright 1999-2022 Gentoo Authors
+# Copyright 1999-2025 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
 EGIT_REPO_URI="https://github.com/digitalocean/${PN}.git"
```

Comment:
`EGIT_REPO_URI` line is usually placed after `HOMEPAGE` variable, could you please move it to the location of now removed `SRC_URI`?